listenbrainz/redis_keys.py

```diff
@@ -1,4 +1,3 @@
 # This module should be used to keep all the of the keys used with redis
 
-INCOMING_QUEUE_SIZE_KEY = "lb.incoming_q_size"
-UNIQUE_QUEUE_SIZE_KEY = "lb.unique_q_size"
+# <crickets> LOL
```

Comment:
I've created an issue for this: https://tickets.metabrainz.org/browse/LB-174

I will do it soon.